decrypt/decrypt.go

```diff
@@ -1,42 +1,213 @@
 package decrypt
 
 import (
+	"errors"
 	"flag"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
 
+	"github.com/NBISweden/sda-cli/helpers"
+	"github.com/elixir-oslo/crypt4gh/keys"
+	"github.com/elixir-oslo/crypt4gh/streaming"
 	log "github.com/sirupsen/logrus"
 )
 
 // Help text and command line flags.
 
+// Usage text that will be displayed as command line help text when using the
+// `help decrypt` command
 var Usage = `
-USAGE: %s decrypt -key <private-key-file> [file(s)]
+USAGE: %s decrypt (-createKey <name>) -key <private-key-file> [file(s)]
```

Comment:
```suggestion
USAGE: %s decrypt -key <private-key-file> [file(s)]
```
CreateKey has no use-case in a decryption scenario